### ISSUE TITLE
use Albumentations 

### ISSUE THREAD

**Issue by futurelee2** [Deci-AI/super-gradients#1845]
### 💡 Your Question

i saw this tutorial
https://github.com/Deci-AI/super-gradients/blob/master/notebooks/albumentations_tutorial.ipynb

but i don't want to use this code 
trainset = COCOFormatDetectionDataset(**train_dataset_params)
trainset.plot()

Can I use 'YoloDarknetFormatDetectionDataset'  instead of the above code to utilize the library?

If I can use it, please let me know how to use it
I wasted too much time :(

**Comment by shaydeci** [comment]
Sure, have you encountered any problems doing so?

**Comment by futurelee2** [comment]
my code is here
input image size is 1280
how can i fix it?

![image](https://github.com/Deci-AI/super-gradients/assets/118868884/9020b321-e83d-4767-bcce-e277d5896c99)
![image](https://github.com/Deci-AI/super-gradients/assets/118868884/e9b02474-22a0-4fb2-9b44-386de899b54b)

**Comment by shaydeci** [comment]
You are using an old SG version that was released before albumentations integration with SG for object detection was introduced.
Please use versions 3.6 and up.
